owrx/controllers/template.py

```diff
@@ -42,4 +43,27 @@ def indexAction(self):
 class MapController(WebpageController):
     def indexAction(self):
         # TODO check if we have a google maps api key first?
-        self.serve_template("map.html", **self.template_variables())
+        # self.serve_template("map.html", **self.template_variables()) # AF: to be removed once the PR is accepted.
+        self.serve_template("map-%s.html" % self.map_type(), **self.template_variables())
+
+    def header_variables(self):
```

Comment:
I believe this part does not work as expected. I don't see the variable `map_type` being referenced in the templates.

Reply:
Apparently because of an incomplete merge. @0xAF , I would look at header.html and also do a global search for "map_type".

Reply:
should be fixed in last commits.